src/Infrastructure/Validator/Manifest/TaskManifestValidator.php

```diff
@@ -0,0 +1,297 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Validator\Manifest;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\ManifestValidatorInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+class TaskManifestValidator implements ManifestValidatorInterface
+{
+    /**
+     * @var string
+     */
+    public const NAME = 'task';
+
+    /**
+     * @var \SprykerSdk\Sdk\Infrastructure\Validator\Manifest\ValidationHelper
+     */
+    protected ValidationHelper $validationHelper;
+
+    /**
+     * @param \SprykerSdk\Sdk\Infrastructure\Validator\Manifest\ValidationHelper $validationHelper
+     */
+    public function __construct(ValidationHelper $validationHelper)
+    {
+        $this->validationHelper = $validationHelper;
+    }
+
+    /**
+     * Returns manifest name.
+     *
+     * @return string
+     */
+    public function getName(): string
+    {
+        return static::NAME;
+    }
+
+    /**
+     * @param string $entityName
+     *
+     * @return \Symfony\Component\Config\Definition\Builder\TreeBuilder
+     */
+    public function getConfigTreeBuilder(string $entityName): TreeBuilder
```

Comment:
Don't need `entityName` there you can use `id` or some generated value

Reply:
will be $config. we can use any key for parent from the structure.